app/backend/src/tests/test_db.py

```diff
@@ -87,6 +88,40 @@ def test_get_parent_node_at_location(testing_communities):
         assert get_parent_node_at_location(session, create_1d_point(51)).id == w_id
 
 
+def test_create_coordinate():
+    longitudes = [
+        (-185.0, 175.0),
+        (-180.0, -180.0),
+        (-175.0, -175.0),
+        (0.0, 0.0),
+        (175.0, 175.0),
+        (180.0, 180.0),
+        (185.0, -175.0)
+    ]
+    latitudes = [
+        (-95.0, -85.0),
+        (-90.0, -90.0),
+        (-85.0, -85.0),
+        (0.0, 0.0),
+        (85.0, 85.0),
+        (90.0, 90.0),
+        (95.0, 85.0)
+    ]
+
+    with session_scope() as session:
+        for lat, lat_expected in latitudes:
+            for lng, lng_expected in longitudes:
+                latitude, longitude = get_coordinates(session.query(create_coordinate(lat, lng)).scalar())
+
+                assert latitude == lat_expected
+
+                # Weird interaction in PostGIS where lng flips at -180 only when there is latitude overflow
+                if lng == -180 and (lat > 90 or lat < -90):
+                    assert longitude == -lng_expected
+                else:
+                    assert longitude == lng_expected
```

Comment:
Just make a list of `lat,lng,lat_exp,lng_exp` and test them with

```py
assert lat_exp, lng_exp == get_coordinates(...)
```

Something like this. Having this logic in the test is not good, now `lng_expected` is not really the expected lng.

Reply:
Yeah, thought about this, thought it might be ok with the comment. 

Changed it now to a few fixed coordinates.